lms/envs/common.py

```diff
@@ -1261,17 +1261,15 @@
     'js/vendor/requirejs/require.js',
     'js/RequireJS-namespace-undefine.js',
     'js/vendor/URI.min.js',
-    'js/vendor/backbone-min.js'
+    'js/vendor/backbone-min.js',
+    'edx-pattern-library/js/modernizr-custom.js',
```

Comment:
Then follow the same pattern as Underscore and JQuery. You will need to add a "defineDependency" line in require-config.js around line 48, and you will need to add the special 'empty:' mapping in build.js, around line 78.